accelerometer/summariseEpoch.py

```diff
@@ -119,10 +121,16 @@ def getActivitySummary(epochFile, nonWearFile, summary,
     # calculate empirical cumulative distribution function of vector magnitudes
     if intensityDistribution:
         calculateECDF(e, 'acc', summary)
-
     # main movement summaries
     writeMovementSummaries(e, labels, summary)
 
+    if psd:
+        calculatePSD(e, summary)
```

Comment:
Can you pass 'labels' into this method?
This would then remove the hard-coding within this method